### PR TITLE
Fix font on narrow screens

### PR DESCRIPTION
**Pull request by erbridge**
### Before

Screenshot 2022-09-22 at 19 42 16

### After

Screenshot 2022-09-22 at 19 41 34

### CODE REVIEW

File: app/views/layouts/_navigation.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="container">
     <a class="navbar-brand hide-desktop" href="/">
       <%= image_tag "mobile-brand.png", width: 30, height: 30, class: "d-inline-block align-text-top" %>
-      ScenicOrNot
+      <span>Scenic</span><span>Or</span><span>Not</span>
```
Comment (snim2) [inline app/views/layouts/_navigation.html.erb:5]: Is this change related to word wrap?

Comment (erbridge) [reply app/views/layouts/_navigation.html.erb:5]: It's so we can style the each word in different colours.

File: app/assets/stylesheets/mobile.sass.scss [hunk 2]
```diff
@@ -102,7 +102,25 @@ $displayfont: "OPTIOliver-Display";
...
+      color: var(--bs-navbar-brand-hover-color);
+    }
+
+    span:nth-of-type(2) {
+      color: $tertiary;
+    }
+
+    span:nth-of-type(3) {
+      color: $primary;
+    }
```
Comment (yndajas) [inline app/assets/stylesheets/mobile.sass.scss:123]: Is there a motivation behind using `nth-of-type` rather than a class to make it more declarative in the stylesheet? (Project doesn't seem very BEM-y, but could still use something to describe what the first, second and third `span`s contain?)

**Review by yndajas — APPROVED** [review]
Looks good, just one suggestion

File: app/assets/stylesheets/mobile.sass.scss (hunk 2 above)
Comment (erbridge) [reply app/assets/stylesheets/mobile.sass.scss:123]: I personally think we often get hung up on that sort of approach. The classes would have no more meaning than `first`, `second`, `third`, so I don't see much value in adding them. It's not like there's some hidden semantics in here, it is really just "colour each word differently".
